docs/conf.py

```diff
@@ -55,4 +56,6 @@
 
 # List of patterns, relative to source directory, that match notebook
 # files that will not be executed.
-nb_execution_excludepatterns = []
+nb_execution_excludepatterns = [
+    'JAX_for_PyTorch_users.ipynb',
```

Comment:
Skipping execution here means that the code in this tutorial will not have any test coverage. I understand this is probably because you showed GPU-specific execution, and we don't have GPUs in our readthedocs build environment. But I wonder if we could structure the discussion in a way that allows for notebook execution – for example mention CUDA support via unexectued code blocks rather than in code cells? What do you think?

Reply:
If we would like to execute this notebook on cpu-only hardware we would need to do the following changes:
- put cells using "cuda" into python code in the text
- add torch, torchvision to requirements.txt
- reduce training time of resnet18 on CIFAR10, 3 epochs -> 1 epoch and 100 iterations per epoch -> 10 iterations per epoch.

Up to you to tell me if we still would like to do that.